components/vocabularies/api/src/main/java/org/phenotips/vocabulary/internal/solr/AbstractSolrVocabulary.java

```diff
@@ -56,6 +56,13 @@
     /** The name of the ID field. */
     protected static final String ID_FIELD_NAME = "id";
 
+    // to "boost" the score of documents in cases where all of the terms in the "q" param appear in close proximity
+    protected static final String COMMON_PARAMS_PF = "pf";
```

Comment:
You should use `DisMaxParams.PF` and `DisMaxParams.QF` instead of redefining them here.